tests/test_Galactic.py

```diff
@@ -76,7 +76,19 @@ def test_proper_motion(self):
             "Applying proper motion for +100d failed with separation %.1e arcsec"
             % sep.arcsec
         )
-        assert sep < 1e-6 * u.arcsec, msg
+        assert sep < 1e-9 * u.arcsec, msg
```

Comment:
Can these be split into separate test cases, so we can see separate failures if something goes wrong?

Reply:
I split the "sanity check" out into its own method.  These other two were originally grouped so I'm inclined to leave them be.